### PR TITLE
semantik: Init at 1.2.5

### PR DESCRIPTION
**Pull request by SCOTT-HAMILTON**
###### Motivation for this change
@blub asked for this package on #nixos IRC

###### Things done
Patched the wscript and tested execution on release 1.2.5

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Blub** [comment]
Different Blub, though :-)

File: pkgs/applications/office/semantik/default.nix [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+, kcoreaddons
+, kconfig
+, kwidgetsaddons
+, ki18n
+, sonnet
+, kdelibs4support
+}:
+mkDerivation rec {
+
+  pname = "semantik";
```
Comment (SuperSandro2000) [inline pkgs/applications/office/semantik/default.nix:27]:
```suggestion
}:

mkDerivation rec {
  pname = "semantik";
```

**Review by SuperSandro2000 — COMMENTED** [review]
@ofborg eval

File: pkgs/applications/office/semantik/default.nix [hunk 2]
```diff
@@ -0,0 +1,91 @@
+{ lib
+, mkDerivation
+, fetchFromGitLab
+, wafHook
+, pkg-config
+, cmake
+, qtbase
+, qttools
```
Comment (SuperSandro2000) [inline pkgs/applications/office/semantik/default.nix:8]:
```suggestion
```

File: pkgs/applications/office/semantik/default.nix [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+}:
+
+mkDerivation rec {
+  pname = "semantik";
+  version = "1.2.5";
+
+  src = fetchFromGitLab {
+    owner = "ita1024";
+    repo = "semantik";
+    rev = "semantik-${version}";
```
Comment (SuperSandro2000) [inline pkgs/applications/office/semantik/default.nix:32]:
```suggestion
    rev = "semantik-${version}";
```

File: pkgs/applications/office/semantik/default.nix [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+      --replace @Qt5Svg@ "${qtsvg}" \
+      --replace @Qt5Svg_dev@ "${lib.getDev qtsvg}" \
+      --replace @Qt5WebEngine@ "${qtwebengine}" \
+      --replace @Qt5WebEngine_dev@ "${lib.getDev qtwebengine}" \
+      --replace /usr/include/KF5/KDELibs4Support "${lib.getDev kdelibs4support}/include/KF5/KDELibs4Support"
+  '';
+
+  nativeBuildInputs = [ (lib.getDev qtsvg) (lib.getLib qtsvg) python3 pkg-config wafHook cmake ];
+
+  buildInputs = [
```
Comment (SuperSandro2000) [inline pkgs/applications/office/semantik/default.nix:61]:
```suggestion
  buildInputs = [
```

**Pull request merged**